packages/features/timezone-buddy/components/AvailabilitySliderTable.tsx

```diff
@@ -78,10 +81,17 @@ export function AvailabilitySliderTable() {
         accessorFn: (data) => data.email,
         header: "Member",
         cell: ({ row }) => {
-          const { username, email, timeZone } = row.original;
+          const { username, email, timeZone, name, organizationId } = row.original;
           return (
             <div className="max-w-64 flex flex-shrink-0 items-center gap-2 overflow-hidden">
-              <Avatar size="sm" alt={username || email} imageSrc={`/${username}/avatar.png`} />
+              <UserAvatar
+                size="sm"
+                user={{
+                  username,
+                  name,
+                  organizationId,
+                }}
+              />
```

Comment:
it seems like we used to display the email as the image alt when the username was not defined, `UserAvatar` doesn't do that anymore, is it a wanted change ?

Reply:
We were doing different things at different places I believe. e.g. I saw that actually user.name is used at most places as `alt` value and if not set empty string is used.

So, I am not sure what we should finalize but to me looks like this is what we could do

§user.name || user.username || user.email§